<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/neon/images/icons/neon-icon-v2.png</normaloff>:/neon/images/icons/neon-icon-v2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="viewStackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menu_Open_recent">
     <property name="title">
      <string>Open &amp;recent</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action_Clear"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="menu_Open_recent"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Snapshot"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_As"/>
   <addaction name="separator"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Undo"/>
   <addaction name="separator"/>
   <addaction name="action_Snapshot"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/document-open.png</normaloff>:/neon/images/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/document-save.png</normaloff>:/neon/images/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/application-exit.png</normaloff>:/neon/images/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Snapshot">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/devices-camera-photo-icon.png</normaloff>:/neon/images/icons/devices-camera-photo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Sna&amp;pshot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/edit-undo.png</normaloff>:/neon/images/icons/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/edit-redo.png</normaloff>:/neon/images/icons/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/help-about.png</normaloff>:/neon/images/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_RegionEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Region Editor</string>
   </property>
  </action>
  <action name="action_ModelSourcesEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Model Sources Editor</string>
   </property>
  </action>
  <action name="action_SceneEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scene Editor</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/document-save-as.png</normaloff>:/neon/images/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="action_SpectrumEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectrum Editor</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/document-new-icon.png</normaloff>:/neon/images/icons/document-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/neon/images/icons/applications-system-icon.png</normaloff>:/neon/images/icons/applications-system-icon.png</iconset>
   </property>
   <property name="text">
    <string>Pr&amp;eferences</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
